tests/dflowfm/test_ext.py

```diff
@@ -584,6 +590,53 @@ def test_given_dict_values_doesnot_raise(self, dict_values: dict):
                     == expected_locationfile
                 )
 
+    class TestMeteo:
+        """Class to test all methods contained in the
+        hydrolib.core.dflowfm.ext.models.Meteo class"""
+
+        def _create_meteo_dict(self) -> Dict:
+            dict_values = {
+                "quantity": "rainfall",
+                "forcingfile": ForcingModel(),
+                "forcingfiletype": MeteoForcingFileType.bcascii,
+                "targetmaskfile": None,
+                "targetmaskinvert": False,
+                "interpolationmethod": None,
+            }
+            return dict_values
+
+        @pytest.mark.parametrize(
+            ("missing_field", "alias_field"),
+            [
+                ("quantity", "quantity"),
+                ("forcingfile", "forcingFile"),
+                ("forcingfiletype", "forcingFileType"),
+            ],
+        )
+        def test_missing_required_fields(self, missing_field, alias_field):
+            dict_values = self._create_meteo_dict()
+            del dict_values[missing_field]
+
+            with pytest.raises(ValidationError) as error:
+                Meteo(**dict_values)
+
+            expected_message = f"{alias_field}\n  field required "
+            assert expected_message in str(error.value)
+
+        def test_construct_from_file(self):
+            input_ext = (
+                test_input_dir
+                / "e02/f006_external_forcing/c069_rain_bc/rainschematic.ext"
```

Comment:
The additional test files (like the dimr.xml) could be removed from the repository. But I will leave it for now since we are going to change our testing approach with regards to test data anyway.

Reply:
I agree!